### PR TITLE
Feature/update slides unit tests

### PR DESCRIPTION
**Pull request by emunib**
## Description

Depends on #95 
Organized existing tests and added unit tests for goToSlide() using the store and router.
## Related Issue or Trello Card

[Trello card](http://www.example.com/)

## Motivation and Context

Update unit test for thoroughness and best practices
## How Has This Been Tested?

- [X] Unit test
- [ ] NightWatch test
- [ ] Coverage testing
- [ ] Manual testing

## Types of changes

- [X] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [X] All new and existing tests passed.

### CODE REVIEW

File: src/store/modules/slide/mutation-types.js [hunk 1]
```diff
@@ -55,3 +55,9 @@ export const SET_META_TIMEOUT = 'SET_META_TIMEOUT'
 export const SET_META_REPEATABLE = 'SET_META_REPEATABLE'
 export const SET_META_STARTDATE = 'SET_META_STARTDATE'
 export const SET_META_ENDDATE = 'SET_META_ENDDATE'
+
+// template which the slide should be displayed
+export const SET_TEMPLATE = 'SET_TEMPLATE'
+
+// timeout of the slide
+export const SET_TIMEOUT = 'SET_TIMEOUT'
```
Comment (KingMackle) [inline src/store/modules/slide/mutation-types.js]: line 58-63 shouldn't be in this PR. We are removing these two mutation types

File: test/unit/specs/Slides.spec.js [hunk 2]
```diff
@@ -1,197 +1,210 @@
 import Vue from 'vue'
+import sinon from 'sinon'
+import Vuex from 'vuex'
+import VueRouter from 'vue-router'
 import Slides from '@/components/Slides'
+import slideModule from '@/store/modules/slide/index'
+import routerModule from '@/router/index'
```
Comment (KingMackle) [inline test/unit/specs/Slides.spec.js]: you don't need to specify `index`, webpack will auto-magically look for it

**Review by KingMackle — CHANGES_REQUESTED** [review]

File: src/store/modules/slide/mutation-types.js (hunk 1 above)
Comment (ehm011) [reply src/store/modules/slide/mutation-types.js]: Done

File: test/unit/specs/Slides.spec.js (hunk 2 above)
Comment (ehm011) [reply test/unit/specs/Slides.spec.js]: Done

**Review by KingMackle — APPROVED** [review]
Looks good, thanks

**Review by phNam2 — APPROVED** [review]

**Review by ert485 — APPROVED** [review]

**Review by Masonfrith — APPROVED** [review]

**Pull request merged**